anna/my-app/src/pages/home/HomePage.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import Layout from '../../components/layout';
+import './homePage.css';
+import MovieList from '../../components/movieList';
+import IMovie from '../../components/movieList/movie-card/IMovie';
+import SearchForm from '../../components/searchForm';
+import { ISearchFormState } from '../../components/searchForm/SearchForm';
+import SortBox from '../../components/sortBox';
+import Container from '../../components/container';
+
+import * as QueryString from "query-string";
+import { RouteComponentProps } from "react-router-dom";
+import Api from '../../Api';
+
+interface IHomeProps {
+  movies: IMovie[];
+  changeMovies: (movies: IMovie[]) => void;
+}
+
+interface IHomeState {
+  sortBy: string;
+}
+
+class HomePage extends React.Component<IHomeProps & RouteComponentProps, IHomeState> {
+
+  state: IHomeState = {
+    sortBy: "release_date"
```

Comment:
You have sortBy in state of two "parent - child" components. Please decide where you wanna handle it. It is a bad approach to manage same state in both components  HomePage and SortBox